core/context.js

```diff
@@ -146,6 +152,15 @@ class TelegrafContext {
     }
   }
 
+  decryptPassportData () {
+    if (this.options.privateKey) {
+      var passport = new TelegramPassport(this.options.privateKey)
+      return passport.decrypt(this.update.message.passport_data)
```

Comment:
Why not `return passport.decrypt(this.passportData)`?

Reply:
I think I wrote this before I added the `passportData` property. I'll update this in the next commit